.github/dependabot.yaml

```diff
@@ -40,6 +40,8 @@ updates:
     # Ignore wrangler
     - dependency-name: "github.com/rancher/wrangler"
     - dependency-name: "github.com/rancher/wrangler/v2"
+    # Ignore Google API version, due it is breaking integration
+    - dependency-name: "google.golang.org/api"
```

Comment:
Looks like this is targeting `release-v2.9` but the PR title mentions `v2.7` and `v2.8`.

Reply:
Thank you, it was my mistake